packages/templates/cra-template-web-viewer/template.json

```diff
@@ -1,58 +1,73 @@
 {
   "package": {
     "dependencies": {
-      "@bentley/icons-generic": "^1.0.13",
-      "@bentley/react-scripts": "^5.0.4",
-      "@itwin/core-telemetry": "^3.7.0",
-      "@itwin/appui-abstract": "^3.7.0",
+      "@bentley/icons-generic": "1.0.13",
+      "@bentley/react-scripts": "5.0.4",
+      "@itwin/appui-abstract": "4.0.0-dev.86",
       "@itwin/appui-layout-react": "rc",
       "@itwin/appui-react": "rc",
-      "@itwin/browser-authorization": "^0.8.0",
+      "@itwin/browser-authorization": "0.8.0",
       "@itwin/components-react": "rc",
-      "@itwin/core-bentley": "^3.7.0",
-      "@itwin/core-common": "^3.7.0",
-      "@itwin/core-frontend": "^3.7.0",
-      "@itwin/core-geometry": "^3.7.0",
-      "@itwin/core-i18n": "^3.7.0",
-      "@itwin/core-markup": "^3.7.0",
-      "@itwin/core-orbitgt": "^3.7.0",
-      "@itwin/core-quantity": "^3.7.0",
+      "@itwin/core-bentley": "4.0.0-dev.86",
+      "@itwin/core-common": "4.0.0-dev.86",
+      "@itwin/core-frontend": "4.0.0-dev.86",
+      "@itwin/core-geometry": "4.0.0-dev.86",
+      "@itwin/core-i18n": "4.0.0-dev.86",
+      "@itwin/core-markup": "4.0.0-dev.86",
+      "@itwin/core-orbitgt": "4.0.0-dev.86",
+      "@itwin/core-quantity": "4.0.0-dev.86",
       "@itwin/core-react": "rc",
+      "@itwin/core-telemetry": "4.0.0-dev.86",
+      "@itwin/ecschema-metadata": "^4.0.0-dev.86",
       "@itwin/imodel-components-react": "rc",
-      "@itwin/imodels-access-frontend": "^3.0.0",
-      "@itwin/itwinui-react": "^2.7.0",
-      "@itwin/presentation-common": "^3.7.0",
-      "@itwin/presentation-frontend": "^3.7.0",
+      "@itwin/imodels-access-frontend": "3.0.0",
+      "@itwin/itwinui-react": "2.7.0",
+      "@itwin/measure-tools-react": "0.11.0-dev.4", 
+      "@itwin/presentation-common": "4.0.0-dev.86",
+      "@itwin/presentation-components": "4.0.0-dev.40",
+      "@itwin/presentation-frontend": "4.0.0-dev.86",
+      "@itwin/property-grid-react": "0.10.0-dev.4",
+      "@itwin/tree-widget-react": "0.9.0-dev.4",
       "@itwin/web-viewer-react": "next",
-      "@itwin/webgl-compatibility": "^3.7.0",
+      "@itwin/webgl-compatibility": "4.0.0-dev.86",
+      "@remix-run/router": "^1.3.2",
       "@types/history": "^4.7.6",
       "@types/node": "^14.0.19",
       "@types/react": "^17.0.9",
       "@types/react-dom": "^17.0.9",
-      "history": "^4.10.1",
+      "history": "^5.3.0",
       "react": "^17.0.2",
       "react-dom": "^17.0.2",
       "react-redux": "^7.2.0",
       "redux": "^4.0.5",
       "typescript": "~5.0.2"
     },
-    "eslintConfig": {
-      "extends": ["react-app", "react-app/jest"]
-    },
     "scripts": {
       "start": "react-scripts --max_old_space_size=4096 start",
       "build": "react-scripts --max_old_space_size=4096 build",
-      "test": "react-scripts --max_old_space_size=4096 test"
+      "test": "react-scripts --max_old_space_size=4096 test",
+      "eject": "react-scripts eject"
+    },
+    "resolutions": {
```

Comment:
this field isnt applicable if youre using pnpm

Reply:
removed field